pom.xml

```diff
@@ -585,12 +585,27 @@
       </dependency>
       <dependency>
         <groupId>org.elasticsearch.plugin</groupId>
+        <artifactId>ingest-attachment</artifactId>
+        <version>${org.elasticsearch.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.elasticsearch.plugin</groupId>
         <artifactId>mapper-attachments</artifactId>
         <version>${org.elasticsearch.version}</version>
       </dependency>
       <dependency>
+        <groupId>org.codelibs.elasticsearch.module</groupId>
```

Comment:
Ah, I 'm ok to do that so.
I will change the groupId and let you upload the artifact in our Nexus.
Thx